t_sparse_list.py

```diff
@@ -109,6 +109,17 @@ def test_slice_reversal_empty(self):
         sl = sparse_list.SparseList(4)
         self.assertEquals([None, None, None, None], sl[::-1])
 
+    def test_slice_with_list_read(self):
+        sl = sparse_list.SparseList([1, 2, 3, 4, 5])
+        ip = [0, 2, 4]
```

Comment:
Ha ! I generally hook my test-suites to display a fortune :cookie: when they pass (you know, positive rewards for self-programmation), and I just had that one :

> Real computer scientists don't comment their code.
> The identifiers are so long they can't afford the disk space.

It's decided : I'll change `ip` to `indices` if you still want to pull this.